test/pls.jl

```diff
@@ -401,6 +401,23 @@ end
     @test countlines(seekstart(io)) == 3
     @test "BlkDiag" in Set(split(String(take!(io)), r"\s+"))
 
+    @testset "optsumJSON" begin
+        fm = last(models(:sleepstudy))
+            # using a IOBuffer for saving JSON
+        saveoptsum(seekstart(io), fm)
+        m = LinearMixedModel(fm.formula, MixedModels.dataset(:sleepstudy))
+        restoreoptsum!(m, seekstart(io))
+        @test loglikelihood(fm) ≈ loglikelihood(m)
+        @test bic(fm) ≈ bic(m)
+        @test coef(fm) ≈ coef(m)
+            # using a temporary file for saving JSON
+        fnm = first(mktemp())
+        saveoptsum(fnm, fm)
+        restoreoptsum!(m, fnm)
```

Comment:
```suggestion
        m = LinearMixedModel(fm.formula, MixedModels.dataset(:sleepstudy))        
        restoreoptsum!(m, fnm)
```

This guarantees that we're not just seeing any lingering state from the earlier tests.